src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -8,6 +8,9 @@
 # pylint: disable=line-too-long
 
 helps['ad sp create-for-rbac'] = """
+            parameters:
+                - name: --cert-name
```

Comment:
That's why it's under the keyvault group.  I really don't like "kv-cert" or any variation of that. I considered overloading "cert" but figured that would be too much.  I could make --create-cert without keyvault honor the cert-name I suppose.